### PR TITLE
Fixing support for `'Elixir.*'` modules.

### PR DESCRIPTION
**Pull request by afronski**
Since that commit `xprof` will support properly *Elixir* applications and modules. 
Of course all related problems are inside *JavaScript*, not *Erlang*. :wink:

**Not ready to merge** - build files are not up to date.

@mniec @gomoripeti @ppikula if someone has version of `webpack` (ideally - `npm ls` so all dependencies together with versions) necessary for building `priv/build/*` somewhere around please send it to me - I'll try to regenerate *build* files then.

After we merge it and tag with new version (I've chosen `1.1.0`) we can publish it in https://hex.pm - this *PR* contains also necessary changes for ready it.

### CODE REVIEW

File: priv/app/function_browser.jsx [hunk 1]
```diff
@@ -107,7 +105,7 @@ export default class FunctionBrowser extends React.Component {
   }
 
   matchFunSignature(input) {
-    var regex = /(\w+):(\w+)\/(\d+)/;
+    var regex = /([\w\.]+):(\w+)\/(\d+)/;
```
Comment (ppikula) [inline priv/app/function_browser.jsx]: if we want to be more strict we can do sth like this: `\w+(\.\w+)*` <-  hope this is a correct regex format :)

Comment (afronski) [inline priv/app/function_browser.jsx]: True, that could be improved - an open question is if we want to care about other valid atom characters and combinations in module name? (e.g. `@` and others).

Comment (ppikula) [inline priv/app/function_browser.jsx]: I'd ignore the other characters until somebody raises an issue. I don't know much about elixir,  but in the erlang world they aren't used frequently

**Comment by ppikula** [comment]
Try with this one, not sure if it going to work though:
https://www.dropbox.com/s/fpz5zc9hbxe1kyt/xprof.txt?dl=0

**Comment by afronski** [comment]
Thanks @ppikula - it helped a lot.

I have managed to fix it and prepare `package.json` from it - so from now on it should work with just `npm install`. Also added fixes for your remark. 😉

**Comment by afronski** [comment]
@mniec **It is ready to merge** - build files are up to date, please tag it after merge with `1.1.0` and we can push it to *Hex*.

**Comment by mniec** [comment]
Woohoo! Awesome work. Thanks!

**Pull request merged**